frontend/public/components/sidebars/resource-sidebars.ts

```diff
@@ -1,9 +1,11 @@
 import { NetworkPolicySidebar } from './network-policy-sidebar';
 import { RoleSidebar } from './role-sidebar';
 import { BuildConfigSidebar } from './build-config-sidebar';
+import { ResourceQuotaSidebar } from './resource-quota-sidebar';
 
 export const resourceSidebars = new Map<string, React.ComponentType<any>>()
   .set('NetworkPolicy', NetworkPolicySidebar)
   .set('Role', RoleSidebar)
   .set('ClusterRole', RoleSidebar)
+  .set('ResourceQuota', ResourceQuotaSidebar)
```

Comment:
nit: not new, but I'd suggest we go ahead and alphabetize this list since it's growing